README.md

```diff
@@ -22,11 +22,14 @@ Open http://localhost:8765 to view it in the browser.
 
 1. Run on a machine with Docker installed:
    ```
-   docker pull cr.yandex/yc/yandex-docker-local-ydb
-   docker run --hostname localhost -e YDB_ALLOW_ORIGIN="http://localhost:3000" -dp 2135:2135 -dp 8765:8765 cr.yandex/yc/yandex-docker-local-ydb
+   docker run --rm -ti --name ydb-local -h localhost \
+      -p 2135:2135 -p 2136:2136 -p 8765:8765 \
+      -v $(pwd)/ydb_certs:/ydb_certs -v $(pwd)/ydb_data:/ydb_data \
+      -e GRPC_TLS_PORT=2135 -e GRPC_PORT=2136 -e MON_PORT=8765 \
+      cr.yandex/yc/yandex-docker-local-ydb:latest
```

Comment:
I think, command here should work without prerequisites. 

Container with such configuration is `unhealthy` on my VM.

Could we use more simple configuration without this row:
```
-v $(pwd)/ydb_certs:/ydb_certs -v $(pwd)/ydb_data:/ydb_data
```

Or, if it's really needed, some description regarding `ydb_certs` and `ydb_data` should be added.

Also this row could be removed, since it contains default values, there is no need to override them for frontend development:
```
-e GRPC_TLS_PORT=2135 -e GRPC_PORT=2136 -e MON_PORT=8765
```

Reply:
Made this command simpler. But `-e MON_PORT=8765` is still there just to match address in `npm run dev` and container port.